entity/entity_pass.h

```diff
@@ -48,7 +48,7 @@ class EntityPass {
 
   EntityPass* GetSuperpass() const;
 
-  bool Render(ContentContext& renderer, RenderPass& parent_pass) const;
+  bool Render(ContentContext& renderer, RenderPass& parent_pass);
```

Comment:
I tried to make this change idempotent so that it's more or less still const from the perspective of the current API (albeit if we were to mark entities and subpasses as `mutable`, that's quite a large surface for problems to arise -- also, if new entities were to be appended between renders, there would be problems). But I get what you're saying here: Since the intention is to keep entities lightweight, I'll change this to go the safer copying route. The performance would also be a bit more consistent that way for really large subpasses.

Reply:
Done. I made this const again and tested with gallery to make sure it's still working.